README.md

```diff
@@ -628,6 +628,23 @@ java -jar modules/swagger-codegen-cli/target/swagger-codegen-cli.jar generate \
   -o samples/server/petstore/jaxrs-cxf
 ```
 
+This Codegen only generate a minimalist server stub. You must add the CXF dependency to your classpath (eg: with Maven)
+If you are using CXF v2.x, you must provided a custom ```ResourceComparator``` class. This class will help CXF to choose the good resource interface for mapping an incomming request. The default behavior of CXF v2.x is not correct when many resources interface have the same global path.
+See: See http://cxf.apache.org/docs/jax-rs-basics.html#JAX-RSBasics-Customselectionbetweenmultipleresources
+
+You can found this class here: https://github.com/hiveship/CXF2-resource-comparator/blob/master/src/main/java/CXFInterfaceComparator.java
+TODO: This class could be directly generated by the Codegen.
+
+You must register this class into your JAX-RS configuration file:
+```xml
+         <jaxrs:resourceComparator>
+   			<bean class="com.akio.cim.aip.rest.utils.CXFInterfaceComparator"/>
+		</jaxrs:resourceComparator> 
+```
+
+This is no longer necessary if you are using CXF >=v3.x
+
```

Comment:
OK thank you. I also updated the file to remove my company name in the 'CXFInterfaceComparator' package. Pease, include it.